README.md

```diff
@@ -7,16 +7,30 @@ A Terraform module for deploying a FreeIPA server.
 ## Usage ##
 
 ```hcl
-module "ipa_master" {
+module "ipa0" {
   source = "github.com/cisagov/freeipa-master-tf-module"
 
-  aws_instance_type   = "t3.large"
-  domain              = "example.com"
-  hostname            = "ipa.example.com"
-  realm               = "EXAMPLE.COM"
-  security_group_ids  = ["sg-51530134", "sg-51530245"]
-  subnet_id           = aws_subnet.master_subnet.id
-  tags                = {
+  domain               = "example.com"
```

Comment:
The source line above show now reference `freeipa-server-tf-module` instead of `freeipa-master-tf-module`.  There are several other references to `freeipa-master-tf-module` in this README that need to be updated.

Reply:
Good catch!  See commit 9ca93d7, which I think catches the remaining references to the old repository name.

Reply:
👍 👍 👍